modules/configuring-the-lora-training-job.adoc

```diff
@@ -0,0 +1,249 @@
+        {"Tweet text":"@mckelldogs This might just be me, but-- eyedrops? Artificial tears are so useful when you're sleep-deprived and sp… https:\/\/t.co\/WRtNsokblG","ID":5,"Label":2,"text_label":"no complaint","output":"### Text: @mckelldogs This might just be me, but-- eyedrops? Artificial tears are so useful when you're sleep-deprived and sp… https:\/\/t.co\/WRtNsokblG\n\n### Label: no complaint"},
+        {"Tweet text":"@Yelp can we get the exact calculations for a business rating (for example if its 4 stars but actually 4.2) or do we use a 3rd party site?","ID":6,"Label":2,"text_label":"no complaint","output":"### Text: @Yelp can we get the exact calculations for a business rating (for example if its 4 stars but actually 4.2) or do we use a 3rd party site?\n\n### Label: no complaint"},
+        {"Tweet text":"@nationalgridus I have no water and the bill is current and paid. Can you do something about this?","ID":7,"Label":1,"text_label":"complaint","output":"### Text: @nationalgridus I have no water and the bill is current and paid. Can you do something about this?\n\n### Label: complaint"},
+        {"Tweet text":"@JenniferTilly Merry Christmas to as well. You get more stunning every year ��","ID":9,"Label":2,"text_label":"no complaint","output":"### Text: @JenniferTilly Merry Christmas to as well. You get more stunning every year ��\n\n### Label: no complaint"}
+    ]
+
+----
+.. Replace the example namespace value `kfto` with the name of your project.
+.. Replace the example training data with your training data.
+.. Save your changes in the `training_data.yaml` file.
+.. Apply the configuration to create the training data:
++
+[source]
+----
+$ oc apply -f training_data.yaml
+----
+
+. Create a persistent volume claim (PVC), as follows:
+.. Create a YAML file named `trainedmodelpvc.yaml`.
+.. Add the following `PersistentVolumeClaim` object definition:
++
+[source]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: trained-model
+  namespace: kfto
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+
+----
+.. Replace the example namespace value `kfto` with the name of your project, and update the other parameters to suit your environment.
+.. Save your changes in the `trainedmodelpvc.yaml` file.
+.. Apply the configuration to create a Persistent Volume Claim (PVC) for the training job:
++
+[source]
+----
+$ oc apply -f trainedmodelpvc.yaml
+----
+
+
+
+
+
+.Verification
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, select your project from the *Project* list. 
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, select your project from the *Project* list.
+endif::[]
+. Click *ConfigMaps* and verify that the `lora-config` and `twitter-complaints` ConfigMaps are listed. 
+. Click *Search*. From the *Resources* list, select *PersistentVolumeClaim* and verify that the `trained-model` PVC is listed.
```

Comment:
Should the sentence on line 239 be broken up into two separate steps?

Reply:
I've rephrased to make it more clear, let me know what you think :)